src/App.js

```diff
@@ -3,12 +3,13 @@ import logo from './logo.svg';
 import './App.css';
 import InputTodo from './InputTodo';
 import TodoList from './TodoList';
+import FilterTodo from './FilterTodo'
 
 class App extends Component {
 
   constructor(props) {
     super(props)
-    this.state = {data: []}
+    this.state = {data: [], filteredData: [], isFiltered: false}
```

Comment:
do we need to keep both data and filtered data in state? The other one can be just stored in class.

Reply:
We were using filteredData for the TodoList component when the filtering is required, as we could not manipulate the data array on every filter

Reply:
We have changed our approach, we have removed filteredData from the state and are using it as a class object in the TodoList component